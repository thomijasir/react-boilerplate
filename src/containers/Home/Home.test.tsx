import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import HomeContainer from './Home.container';

let wrapper;
// let instance;

beforeEach(() => {
  wrapper = shallow(<HomeContainer />, {
    disableLifecycleMethods: false,
  });
  // instance = wrapper.instance();
});

afterEach(() => {
  jest.clearAllMocks();
});

describe('Home Snap Test', () => {
  describe('render()', () => {
    test('renders the component', () => {
      wrapper = shallow(<HomeContainer />);
      expect(toJson(wrapper)).toMatchSnapshot();
    });
  });
});
