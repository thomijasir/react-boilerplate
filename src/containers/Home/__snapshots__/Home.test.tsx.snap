// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Snap Test render() renders the component 1`] = `
<div
  className="home-page"
>
  <div
    className="container-fluid"
  >
    <div
      className="row"
    >
      <div
        className="col-lg-12"
      >
        <img
          alt="react-logo"
          className="rotate"
          src="react.svg"
        />
        <h1>
          React Boilerplate
        </h1>
        <p>
          This template project was designed for easy scalability apps.
        </p>
        <small>
          Architecture by Thomi Jasir
        </small>
      </div>
    </div>
  </div>
</div>
`;
